tools/Linux/kodi.sh.in

```diff
@@ -208,6 +214,8 @@ do
   [ -f "${APPORT_CORE}" ] && rm -f "${APPORT_CORE}"
   LOOP=0
   ${KODI_BINARY} $SAVED_ARGS
+  CHILD=$!
+  wait "${CHILD}"
```

Comment:
for this to work kodi need to be started in background.

Reply:
of course, yes -- missed that when copying from my local version into the repo